packages/fluxible-plugin-perf/lib/perf-plugin.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * Copyright 2014, Yahoo! Inc.
+ * Copyrights licensed under the New BSD License. See the accompanying LICENSE file for terms.
+ */
+'use strict';
+var debug = require('debug')('Fluxible:PerfPlugin');
+var GLOBAL_UUID_MAX = Math.pow(2, 53);
+function generateUUID () {
+    return Math.ceil(Math.random() * GLOBAL_UUID_MAX);
+}
+
+/**
+ * Creates a new fetchr plugin instance with options
+ * @param {Object} options configuration options
+ * @returns {PerfPlugin}
+ */
+module.exports = function perfPlugin(options) {
+    debug('initialized');
+    /**
+     * @class PerfPlugin
+     */
+    return {
+        /**
+         * @property {String} name Name of the plugin
+         */
+        name: 'fPerfPlugin',
+        /**
+         * Called to plug the FluxContext
+         * @method plugContext
+         * @param {Object} contextOptions options passed to the createContext method
+         * @param {Object} contextOptions.debug if true, will enable debug mode and allow fluxible to collect metadata for debugging
+         * @returns {Object}
+         */
+        plugContext: function plugContext(contextOptions, context) {
+            debug('plugged', arguments);
+            var enableDebug = typeof contextOptions.debug !== 'undefined' ? contextOptions.debug : false;
+            var actionHistory = [];
+            console.log('perf enabled', enableDebug);
+            console.log(context);
+            context.getActionHistory = function () {
```

Comment:
I suggest we namespace the function with `context.perfTools.` (or `context.devTools.`)